.changeset/red-pugs-promise.md

```diff
@@ -0,0 +1,32 @@
+---
+'@lion/ui': patch
+---
+
+This introduces a new package `@lion/ui` which is a collection of UI components that can be used in your application. It contains all the components/system that used to be distributed via separate `@lion/*` packages.
```

Comment:
```suggestion
This introduces a new package `@lion/ui` which is a collection of UI components that can be used in your application. It contains all the components/systems that used to be distributed via separate `@lion/*` packages.
```

Reply:
this will start a new changelog for a new package... I don't think it makes sense to start a new release with "breaking"...